dev/tools/dartdoc.dart

```diff
@@ -52,7 +53,15 @@ dependencies:
   if (code != 0)
     exit(code);
 
-  // Generate the documentation; we require dartdoc >= 0.9.4.
+  // Install dartdoc.
```

Comment:
I'll update w/ some text.

It may still be useful to peg at a specific version, for reproducibility. We may want to use bin/cache/dart-sdk/bin/dartdoc for that however.